geoh5py/ui_json/input_file.py

```diff
@@ -130,6 +129,17 @@ def data(self, value: dict[str, Any]):
 
         self._data = value
 
+    @property
+    def plain_data(self) -> dict[str, Any] | None:
```

Comment:
@benk-mira , I had to introduce that function for using in param-sweeps, so that it give out a path and not a workspace in the dictionary values

Reply:
see usage here: https://github.com/MiraGeoscience/param-sweeps/pull/20/files#diff-abc62fce21529d9bfb8e25b1b4a47d887d40d70a580828f8f0600e5b03f161f6R47